cmd/common.go

```diff
@@ -121,27 +121,60 @@ func listRoles(ctx *cli.Context, context string) error {
 	return writer.Err()
 }
 
-func listRoleTemplateBindings(ctx *cli.Context, b []RoleTemplateBinding) error {
-	writer := NewTableWriter([][]string{
+func listRoleTemplateBindings(ctx *cli.Context, writerConfig *TableWriterConfig, rtbs []RoleTemplateBinding) error {
+	writer := NewTableWriterWithConfig([][]string{
 		{"BINDING-ID", "ID"},
-		{"USER", "User"},
+		{"MEMBER", "Member"},
 		{"ROLE", "Role"},
 		{"CREATED", "Created"},
-	}, ctx)
-
+	}, writerConfig)
 	defer writer.Close()
 
-	for _, item := range b {
-		writer.Write(&RoleTemplateBinding{
-			ID:      item.ID,
-			User:    item.User,
-			Role:    item.Role,
-			Created: item.Created,
-		})
+	for _, rtb := range rtbs {
+		writer.Write(&rtb)
 	}
+
 	return writer.Err()
 }
 
+type principalGetter interface {
+	ByID(id string) (*managementClient.Principal, error)
+}
+
+func getMemberNameFromPrincipal(principals principalGetter, principalID string) string {
+	principal, err := principals.ByID(url.PathEscape(principalID))
+	if err != nil {
+		principal = parsePrincipalID(principalID)
```

Comment:
Should this maybe only happen if it's not found? Feels a little odd to show the user details if we got a network error communicating with the cluster?